packages/components/src/entities/SampleActionsButton.tsx

```diff
@@ -23,16 +23,18 @@ import { PicklistCreationMenuItem } from '../internal/components/picklist/Pickli
 import { AddToPicklistMenuItem } from '../internal/components/picklist/AddToPicklistMenuItem';
 
 import { AssayResultsForSamplesMenuItem } from './AssayResultsForSamplesButton';
+import { FindDerivativesMenuItem } from './FindDerivativesButton';
 
 interface Props {
     disabled?: boolean;
     metricFeatureArea?: string;
     model: QueryModel;
+    sampleFinderProps?: Record<string, any>;
```

Comment:
both the LKB and LKSM implementations are passing in the same value for this prop (i.e. `AssayResultDataType`) and that value is already in the components package. I don't see any other usages of `SampleActionsButton` so it seems like this prop can be dropped and the entityDataType inlined below at line 53.

Reply:
The passed in props now also contain baseFilters, so kept them as Record<string, any>